Rework installation steps (#227)
estkme-group/lpac: docs/DEVELOPERS.md

### macOS

Install [Homebrew](https://brew.sh/).  
Execute the same commands as you would do in Debian.  

---

### Windows(x86_64)

Windows needs libcurl.dll to run.  

> Download libcurl from <https://curl.se/download.html> and place it as `libcurl.dll` in the `build/output` directory.  

Reviewer: ```suggestion
Download libcurl from <https://curl.se/download.html> and place it as `libcurl.dll` aside `lpac.exe`.  
```
More accuracy.
